### DISCUSSION TITLE
Responses should be an array, not a map

### DISCUSSION THREAD

**Discussion by daniel-white** [OAI/sig-moonwalk#83] [Ideas]
In order to promote reuse of responses and reducing the duplication of the response key, the response object should be an array of responses that have a name property.

Instead of
```yaml
responses:
  ok:
    contentType: application/json
  badRequest:
    $ref: '#/components/responses/badRequest'
```

Consider

```yaml
responses:
  - name: ok
    contentType: application/json
  - $ref: '#/components/responses/badRequest'
```

**Comment by handrews** [comment]
Thanks @daniel-white - this is a duplicate of #25 so I'm going to close it, but please feel free to comment there, or on the more general object vs array discussion in #32.
